src/yt/chat.ts

```diff
@@ -20,7 +20,10 @@ export interface YTTextRun {
   text: string;
   bold?: boolean;
   italics?: boolean;
-  navigationEndpoint?: YTUrlEndpointContainer | YTBrowseEndpointContainer;
+  navigationEndpoint?:
+    | YTUrlEndpointContainer
+    | YTBrowseEndpointContainer
+    | YTWatchEndpointContainer;
```

Comment:
When do you get WatchEndpoint? :InaThink:

Reply:
There is a sample from RBC's livechat:
https://www.youtube.com/watch?v=CJUAYSAVtic&t=557s

```json
{
  "runs": [
    {
      "text": "https://youtu.be/5A_RqpENxvg",
      "navigationEndpoint": {
        "clickTrackingParams": "CAEQl98BIhMImPXK9t6Z8wIVjFcqCh1KbwuN",
        "commandMetadata": {
          "webCommandMetadata": {
            "url": "/watch?v=5A_RqpENxvg",
            "webPageType": "WEB_PAGE_TYPE_WATCH",
            "rootVe": 3832
          }
        },
        "watchEndpoint": {
          "videoId": "5A_RqpENxvg",
          "nofollow": true
        }
      }
    }
  ]
}
```

but it also includes `YTUrlEndpointContainer`
maybe no need to handle it here right...?